includes/multi-currency/client/settings/single-currency/__tests__/index.test.js

```diff
@@ -170,9 +170,6 @@ describe( 'Single currency settings screen', () => {
 		expect(
 			screen.queryByText( /Currency Settings/i )
 		).toBeInTheDocument();
-		expect(
-			screen.queryByText( /Current rate: 1 USD = 0.826381 EUR/ )
-		).toBeInTheDocument();
```

Comment:
This text is no longer being generated in the snapshot because the version of WP components generating this snapshot did not include the help text.